source/fundamentals/indexes.txt

```diff
@@ -205,8 +216,11 @@ database is a GeoJSON Point object that describes the coordinates of the theater
       }
    }
 
-The following example uses the ``createIndexes()`` method to create a ``2dsphere`` index on the ``location.geo`` field
-in the ``theaters`` collection in the ``sample_mflix`` database to enable geospatial searches.
+The following example creates a ``2dsphere`` index on the ``location.geo`` field:
+
+.. note::
+
+   Attemping to create a geospatial index on a field that is covered by a geospatial index will result in an error.
```

Comment:
Is this a common enough problem to warrant a note? To me, this seems intuitive enough to omit, but maybe I'm missing something? If it helps even a small number of users, I think it's worth keeping, though.

Reply:
So, the index story in MongoDB isn't uniform. The other index types that I didn't wrap in a try/catch can be created over and over on the same field, MongoDB will just happily return the name of the existing index.

These other types throw an error. Since I've wrapped the operation in a try/catch I thought it best to add a note.

Reply:
That sounds like a norm-defying footgun to me. LGTM.